### PR TITLE
Add more error checking to the publish form.

### PR DESCRIPTION
**Pull request by laceysanderson**
# Bug Fix
Issue #852 

## Description
Adds a little more error checking to the lines causing the error reported in #852. The original issue is not reproducible but the error checking and default I added are pretty generally applicable.

## Testing?
Try to publish content and ensure I haven't broken the form ;-p Other then that, a code review should be sufficient.

### CODE REVIEW

**Comment by bradfordcondon** [comment]
publish page now crashes

```
ParseError: syntax error, unexpected '{' in menu_execute_active_handler() (line 93 of /Users/bc/tripal/sites/all/modules/tripal/tripal_chado/includes/tripal_chado.publish.inc).
```

looks like a missed brace

File: tripal_chado/includes/tripal_chado.publish.inc [hunk 1]
```diff
@@ -90,10 +90,22 @@ function tripal_chado_publish_form($form, &$form_state) {
 
         // Now call the widget callback function to let the module adjust the
         // element as needed.
+        if (!isset($instance['widget']['module'](( {
```
Comment (almasaeed2010) [inline tripal_chado/includes/tripal_chado.publish.inc]:
```suggestion
        if (!isset($instance['widget']['module'])) {
```

Comment (almasaeed2010) [reply tripal_chado/includes/tripal_chado.publish.inc]: @bradfordcondon you should try again with this fix

Comment (bradfordcondon) [inline tripal_chado/includes/tripal_chado.publish.inc]: Parenthesis in wrong direction 😈 
```suggestion
        if (!isset($instance['widget']['module'])) {
```

Comment (bradfordcondon) [reply tripal_chado/includes/tripal_chado.publish.inc]: yep agreed

**Comment by spficklin** [comment]
This worked fine for me when testing. I understand the code changes, and I see the fix @bradfordcondon made.  I think this is good and can be merged.   Travis CI is failing but on the custom quota and not related to this fix, so I'm merging.

**Pull request merged**